src/main/api/2/studio-api-2.yaml

```diff
@@ -3765,10 +3765,11 @@ paths:
 
   /content/children_by_path:
     post:
+      deprecated: true
       tags:
         - content
       summary: Get list of children for given item path
-      description: 'Required permission "get_children" and "site member"'
+      description: 'Required permission "get_children" and "site member".<br/> This operation is deprecated, use /content/{siteId}/children instead.'
```

Comment:
Add `deprecated: true` instead of the comment.

Reply:
`deprecated: true` is already there. Added the comment to indicate the alternative API endpoint